src/coldfront_plugin_api/utils.py

```diff
@@ -55,4 +55,11 @@ def get_or_fetch_user(username):
 
 
 def is_user_superuser(user: User):
-    return user.is_superuser
+    """
```

Comment:
Can you please describe why this is necessary? What error code did it return as only `return user.is_superuser` and what does it do now.

Reply:
@QuanMPhm ping about responding to this comment.